libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -984,6 +984,7 @@ IotMqttError_t IotMqtt_Init( void )
          * condition is not an invariant, and the MISRA 14.3 violation is
          * a false positive. */
         /* coverity[misra_c_2012_rule_14_3_violation] */
+        /* coverity[const] */
```

Comment:
Why const exception is needed ? Same reason as 14.3 ?

Reply:
Yes, there were two events for that violation:
```
(1) Event const: | At condition "status == IOT_MQTT_SUCCESS", the value of "status" must be equal to 0.
(3) Event misra_c_2012_rule_14_3_violation: | The condition "status == IOT_MQTT_SUCCESS" must be true.
```